bin/install-qa-check.d/60pkgconfig

```diff
@@ -1,12 +1,70 @@
 # Check for pkg-config file issues
 
 pkgconfig_check() {
+	# Make a list of .pc files and bail out if there aren't any
+	mapfile -d '' files < <(find "${ED}"/usr/lib*/pkgconfig "${ED}"/usr/share/pkgconfig -maxdepth 1 -type f -name '*.pc' -print0 2>/dev/null)
+	[[ -n "${files[@]}" ]] || return
```

Comment:
```suggestion
	[[ -z "${files[@]}" ]] && return
```

Generally people find it easier to read the logic without negations, i.e. 'if X then Y'.